Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py

```diff
@@ -0,0 +1,611 @@
+                    else:
+                        table_values.append(item)
+                table_contents[key] = table_values
+            else:
+                table_contents[key] = val
+        return table_contents
+    except Exception as e:
+        raise DemistoException(e)
+
+
+def post_submit_file(client: Client, args: Dict[str, str]) -> CommandResults:
+    urlSuffix = "/submit_file"
+    entryId = args.get('EntryID') or None
+    if entryId is None:
+        raise DemistoException("Entry ID Not Found")
+    platformValue = args.get('platform') or SNXResponse.SNX_WINDOWS_KEY
+    params = create_request_json(args)
+    if 'platform' not in params.keys():
+        params['platform'] = platformValue
+    fileEntry = demisto.getFilePath(entryId)
+    fileName = fileEntry['name']
+    filePath = fileEntry['path']
+    fileData = {'file': (fileName, open(filePath, 'rb'))}
+    response, err_msg = client.submit_file(urlSuffix, fileData, params)
+    if response:
+        if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
+            finalJson = response[SNXResponse.SNX_RESULT_KEY]
+            readableOutput = tableToMarkdown(f"File Submitted Successfully: {fileName}", finalJson)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="SnXAnalysis.SubmitFile",
+                outputs=finalJson
+            )
+        else:
+            readableOutput = tableToMarkdown("File Submission Failed: {fileName}", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
+    else:
+        msg = error_response(err_msg)
+        raise DemistoException(msg)
+
+
+def post_submit_url(client: Client, args: Dict[str, str]) -> CommandResults:
+    urlSuffix = "/analyze_url"
+    urlValue = args.get("URL") or None
+    if urlValue is None or len(urlValue) == 0:
+        raise DemistoException("Input url value is empty")
+    params = create_request_json(args)
+    urlParams = {SNXReportParser.JSON_URL: urlValue}
+    response, err_msg = client.submit_url(urlSuffix, params, urlParams)
+    if response:
+        if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
+            finalJson = response[SNXResponse.SNX_RESULT_KEY]
+            readableOutput = tableToMarkdown("URL Submitted Successfuly", finalJson)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="SnXAnalysis.SubmitURL",
+                outputs=finalJson
+            )
+        else:
+            readableOutput = tableToMarkdown("URL Submission Failed", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
```

Comment:
This `outputs` will not be under a special key.
Consider removing `outputs` so it doesn't return to context, or add to `output_prefix` like this:

```suggestion
            return CommandResults(
                readable_output=readableOutput,
                outputs_prefix="SnXAnalysis.SubmitURL",
                outputs=response
            )
```